### PR TITLE
set committer author for digestbot updates

### PR DESCRIPTION
**Pull request by cpanato**

### CODE REVIEW

File: .github/workflows/digestabot.yaml [hunk 1]
```diff
@@ -27,3 +27,5 @@ jobs:
     - uses: chainguard-dev/digestabot@4298e530896fb4164f9c89030c561b75b9321149 # v1.0.2
       with:
         token: ${{ steps.octo-sts.outputs.token }}
+        author: chainguardian
+        committer: "101908552+chainguardian@users.noreply.github.com"
```
Comment (xnox) [inline .github/workflows/digestabot.yaml:31]: Hm, given these pull requests are done by octo-sts bot, I would want to see octo-sts bot attribution here.

I'm not sure what is its author and commiter name, but would this work?

```suggestion
        author: "octo-sts[bot]"
        committer: "157150467+octo-sts[bot]@users.noreply.github.com"
```

Comment (cpanato) [reply .github/workflows/digestabot.yaml:31]: sgtm

Comment (cpanato) [reply .github/workflows/digestabot.yaml]: thanks! PTAL

**Comment by xnox** [comment]
Let's see if I got the author name and id right.

**Review by xnox — APPROVED** [review]

**Comment by cpanato** [comment]
that is correct

**Review by rawlingsj — APPROVED** [review]

**Pull request merged**